stdlib/universe/table_fns_test.go

```diff
@@ -139,7 +139,7 @@ func TestTableFind_Call(t *testing.T) {
 		want flux.Table
 		fn   string
 		// fn      func(key values.Object) (values.Value, error)
-		wantErr error
+		wantErr      error
```

Comment:
added by `go fmt`

Reply:
This was my fault. I updated a PR and and push forced, but forgot to run  `make fmt`. Not sure how it passed CI though ... 

Anyhow, I included a fix for this in the PR I've got open. #2583. I'm not sure if that's acceptable, to piggyback some whitespace changes on top of another patch, so I can fix that if necessary.

Reply:
That's fine with me. I can just wait for your change to get merged, reset my formatting and rebase off of your changes.